e2e/nextjs/.env

```diff
@@ -1,3 +1,3 @@
 NEXT_PUBLIC_HIGHLIGHT_PROJECT_ID='1jdkoe52'
-NEXT_PUBLIC_HIGHLIGHT_OTLP_ENDPOINT='http://localhost:4318'
-NEXT_PUBLIC_HIGHLIGHT_BACKEND_URL='https://localhost:8082/public'
\ No newline at end of file
+# NEXT_PUBLIC_HIGHLIGHT_OTLP_ENDPOINT='http://localhost:4318'
```

Comment:
This change isn't necessary, but I prefer to set these values in my `env.local` so that the `e2e/nextjs` demo project defaults to sending errors to prod.